src/session.rs

```diff
@@ -9,7 +13,7 @@ pub struct SessionOptions {
 
 #[napi]
 pub struct SessionWrapper {
-    internal_session: Session,
+    internal: Session,
 }
```

Comment:
Such a rename should be done in a separate commit, not to clutter the commit that contains meaningful new logic.